src/lib/routing-tables.js

```diff
@@ -56,7 +56,7 @@ class RoutingTables {
    * @param {Route|RouteData} _route from ledger B→C
    * @returns {Boolean} whether or not a new route was added
    */
-  addRoute (_route) {
+  addRoute (_route, prefix) {
```

Comment:
It might also be simpler to add it as a new property of `Route` rather than passing it around separately.